bedrock/releasenotes/models.py

```diff
@@ -88,8 +88,7 @@ def extendMarkdown(self, md):
 
 
 def process_markdown(value):
-    rendered_html = markdowner.reset().convert(value)
-    return bleach.clean(rendered_html, tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRS)
+    return markdowner.reset().convert(bleach.clean(value, tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRS))
```

Comment:
This reverts the change made [here](https://github.com/mozilla/bedrock/pull/12493/commits/b29693789ecab21938fc24cc49f56dc8874bd2a2) which overlooked the use of raw HTML in release-notes Markdown.

Another option for a fix would have been to expand the allowlist, but this approach reinstates expected behaviour more reliably and quickly

Reply:
I do think it'd be better to expand the allow list. I'm not sure we can fully trust a markdown renderer to never produce anything shady. Not urgent to fix though I'd say.

Reply:
Ticketed for careful follow-up here https://github.com/mozilla/bedrock/issues/12560